packages/loader/container-loader/src/deltaManager.ts

```diff
@@ -218,7 +218,7 @@ export class DeltaManager<TConnectionManager extends IConnectionManager>
             return -1;
         }
 
-        if (contents !== undefined) {
+        if (contents !== undefined && contents !== null) {
```

Comment:
I'd change that to
> if (contents)

It includes undefined, null and "", which are all fine in this context (though comment might be nice to add on null & undefined)